src/schemata/string/Base64.ts

```diff
@@ -64,7 +64,7 @@ export const Base64: Base64S = make(S =>
     S.padRight(
       {
         by: 'ExactLength',
-        exactLength: s => (s.length === 0 ? 0 : s.length + (4 - (s.length % 4))),
+        exactLength: s => s.length + ((4 - (s.length % 4)) % 4),
```

Comment:
important part of changes here ^

`(4 - (s.length % 4))` makes it 1, 2, or 4 instead of 1, 2, or 0, so it was always failing when there should've been no padding because it was expecting 4 =s

Reply:
No worries!  That extra mod 4 is tricky

Reply:
Is that the same as mod 3?

Reply:
no, because `4 % 3 === 1`

Reply:
I feel like I tried literally everything but 3